resource/example.json

```diff
@@ -9,7 +9,7 @@
 		{
 			"warehouse_id": "15566",
 			"shipment_method": "truck",
-			"Shipment_id": "1adf4",
+			"shipment_id": "1adf4",
```

Comment:
Last week when I asked if we could modify the JSON file, he disapprove.
IDK if this is a typo or done purposefully.

but in the case that we are not to touch the JSON,

id = shipmentObject.get("shipment_id");
if (id == null)
  id = shipmentObject.get("Shipment_id");